### PR TITLE
PAS storage buff

### PR DESCRIPTION
**Pull request by lovegreenstuff**
## About The Pull Request
Brings PAS internal storage to the same level as jaeger general storage module.

## Why It's Good For The Game
Me like competitive armor option.
Me want hold flamer tank in PAS vest for funny underbarrel.

## Changelog
:cl:
balance: increased weight limit for pas storage.
/:cl:

### CODE REVIEW

**Comment by lovegreenstuff** [comment]
yes this is basically a carbon copy of a previous PR, but it actually works this time.
the things that you learn when you actually test your code.

**Comment by TiviPlus** [comment]
> jaegercucks

No cussing

**Comment by lovegreenstuff** [comment]
fine

File: code/modules/clothing/suits/marine_armor.dm [hunk 1]
```diff
@@ -273,6 +273,10 @@
 	soft_armor = list("melee" = 40, "bullet" = 60, "laser" = 60, "energy" = 45, "bomb" = 45, "bio" = 45, "rad" = 45, "fire" = 45, "acid" = 50)
 	slowdown = 0.5 //a bit less
 	light_range = 6
+	pockets = /obj/item/storage/internal/suit/marine/pasvest
+
+/obj/item/storage/internal/suit/marine/pasvest
+	max_w_class = 3
```
Comment (QualityVan) [inline code/modules/clothing/suits/marine_armor.dm:279]:
```suggestion
	max_w_class = WEIGHT_CLASS_NORMAL
```

Comment (lovegreenstuff) [reply code/modules/clothing/suits/marine_armor.dm:279]: oh right readability

**Comment by BraveMole** [comment]
PAS doesn't need that, this is powercrepe

**Pull request closed without merging**